app/views/docs/permissions.phtml

```diff
@@ -4,15 +4,15 @@
 
 <p>As the name suggests, read permission allows a user to view a resource, create allows users to create new resources, update allows a user to make changes to a resource, and delete allows the user to remove the resource.</p>
 
-<p>All permissions can be given to specific users, entire teams, or only to team members with a specific role.</p>
+<p>All permissions can be granted to inividual or groups of users, entire teams, or only to team members with a specific role. Permission can also be granted based on authentication status such as to all users, only authenticated users, or only guest users.</p>
 
-<p>A project user can only grant a resource with permissions that they own. For example, if a user is trying to share a document with a team he or she are not members of, they will encounter a 401 not authorized error. If your app needs users to grant access to teams they're not a member of, you can create Appwrite Functions with a <a href="/docs/getting-started-for-server">Server SDK</a> to achieve this functionality.</p>
+<p>A project user can only grant a resource with permissions that they own. For example, if a user is trying to share a document with a team that they are not a member of, they will encounter a 401 not authorized error. If your app needs users to grant access to teams they're not a member of, you can create Appwrite Functions with a <a href="/docs/getting-started-for-server">Server SDK</a> to achieve this functionality.</p>
 
 <div class="notice calm margin-top-large margin-bottom-large">
     <h4>Appwrite Resource</h4>
     
-    <p>An Appwrite resource can be a database, collection, document, bucket, or a file. Each resource can have a set of permissions that govern the type access granted to users.</p>
-    
+    <p>An Appwrite resource can be a database, collection, document, bucket, or a file. Each resource has read, write, update, and delete permissions to define who can interact with it.</p>
```

Comment:
```suggestion
    <p>An Appwrite resource can be a database, collection, document, bucket, or a file. Each resource has create, read, update, and delete permissions to define who can interact with it.</p>
```

Reply:
We need to mention that `documents` and `files` only have `read`, `update`, `delete` permissions. 
All the other resources have a `create` permission in addition to the former.

Reply:
>We need to mention that documents and files only have read, update, delete permissions.
All the other resources have a create permission in addition to the former.

I will address this in our list of query methods and leave the specific methods out of the note to avoid redundant information + the note getting to long.